lib/volume.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "pathname"
+
+# A named storage volume for a given format. This assumes for now that volumes
+# are accessible via normal disk (File/IO) operations.
+class Volume
```

Comment:
It makes me a little bit itchy to have this and `VolumeManager` at the top level namespace rather than in `Chipmunk::`, especially as `VolumeManager` raises a `Chipmunk::VolumeNotFoundError`.

Reply:
Per discussion, we agree to move these in. Some of this terminology is overloaded and we don't want to imply that this is an enterprise-wide notion of what a volume is. It's a volume _for Chipmunk_, so we should namespace it thus.

In DDD terms, this is emerging as a bounded context for artifact ingest and preservation. I do sorely wish that we had better packaging/namespace/import support in Ruby...